lib/redis-store.js

```diff
@@ -47,9 +50,9 @@ var RedisStore = function(options) {
         }
 
         replies = processReplies(replies);
-        setExpire(replies, rdskey);
+        var ttl = setExpire(replies, rdskey);
 
-        cb(null, replies[0]);
+				cb(null, replies[0], ttl > 0 ? new Date(new Date().getTime() + ttl) : null);
```

Comment:
It looks like there was unintentional spacing added to this, if you can address it, it can get merged.

Reply:
I'have fix spaces.
Thanks